src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -177,11 +183,17 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem, Cancellatio
             var loadCts = CancellationTokenSource.CreateLinkedTokenSource(token);
             Interlocked.Exchange(ref _loadCts, loadCts)?.Cancel();
 
+            // add Loading... indicator if not present
```

Comment:
comment doesn't match code.

Reply:
Changed to correct meaning: add the loading indicator at the beginning.